src/recon_buildblock/GeneralisedObjectiveFunction.cxx

```diff
@@ -47,6 +50,7 @@ GeneralisedObjectiveFunction<TargetT>::
 initialise_keymap()
 {
   this->parser.add_parsing_key("prior type", &prior_sptr);
+  this->parser.add_key("Use TOF information", &use_tof);
```

Comment:
need to remove. This class knows nothing about the data, so TOF makes no sense here.. The user can always create a non-tof sino (harder with listmode recon of course).

Reply:
Is there documentation detailing how this is done?